app.js

```diff
@@ -0,0 +1,18 @@
+const express  = require('express')
+const app = express()
+const {getCategories} = require('./controllers/categories.controller.js')
+
+app.use(express.json())
+
+/* /api/categories */
+app.get('/api/categories', getCategories)
+
+
+
+
+app.use((err, req, res, next) => {
+    res.status(400).send({msg: 'Bad REquest', error: err})
+})
+
```

Comment:
Could we add some error handling middleware for the 404 please? We'd also like to see a 500 error in here too 👍